src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/MeetTheSpecialists.tsx

```diff
@@ -61,16 +73,15 @@ export const MeetTheSpecialists: React.FC = () => {
       <Text mb={2} variant={["xs", "sm"]}>
         Our specialists span today’s most popular collecting categories.
       </Text>
-      <Rail
-        title=""
-        getItems={() => {
-          return SPECIALISTS.map(i => (
+      <Shelf showProgress={false}>
+        {specialists.map(i => {
```

Comment:
**nitpick**: could you rename `i` to `specialist`? It's very minor, but variables like that (especially `i`) may be very confusing

Reply:
Sure 👍